### PR TITLE
[4] Improve Redis configuration field help/label

### PR DESCRIPTION
**Pull request by PhilETaylor**
additional, but standalone PR to #26269 and #33880

### Summary of Changes

Visual text changes only.

Add description to show that port is discarded when using a unix socket connection (As that is what Joomla does, but seeing 6379 would be confusing for someone with a Redis background like me and others @simbus82)

Add `/Sock` to host label to show that Joomla supports Redis Socket connections and that you can enter a tcp Host or a unix socket path. 

Screenshot 2021-05-14 at 20 49 45

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 6c062ac20ed973c8ebfee2fefc627dbe23dd3df6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33882">issues.joomla.org/tracker/joomla-cms/33882</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6c062ac20ed973c8ebfee2fefc627dbe23dd3df6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33882">issues.joomla.org/tracker/joomla-cms/33882</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33882">issues.joomla.org/tracker/joomla-cms/33882</a>.</sub>

File: administrator/language/en-GB/com_config.ini [hunk 1]
```diff
@@ -136,8 +136,9 @@ COM_CONFIG_FIELD_PROXY_PORT_LABEL="Outbound Proxy Port"
 COM_CONFIG_FIELD_PROXY_USERNAME_LABEL="Outbound Proxy Username"
 COM_CONFIG_FIELD_REDIS_AUTH_LABEL="Redis Server Authentication"
 COM_CONFIG_FIELD_REDIS_DB_LABEL="Redis Database"
-COM_CONFIG_FIELD_REDIS_HOST_LABEL="Redis Server Host"
+COM_CONFIG_FIELD_REDIS_HOST_LABEL="Redis Server Host/Sock"
 COM_CONFIG_FIELD_REDIS_PERSISTENT_LABEL="Persistent Redis"
+COM_CONFIG_FIELD_REDIS_PORT_DESC="Port discarded if connecting to Redis server over unix sockets."
```
Comment (richard67) [inline administrator/language/en-GB/com_config.ini]:
```suggestion
COM_CONFIG_FIELD_REDIS_PORT_DESC="Port discarded if connecting to Redis server over Unix sockets."
```

**Comment by richard67** [comment]
Thanks!

**Pull request merged**